index.js

```diff
@@ -13,6 +13,12 @@ import PropTypes from 'prop-types'
 
 const { UIManager } = NativeModules
 
+export const TextTickAnimationType = Object.freeze({
+  auto: 'auto',
+  scroll: 'scroll',
+  bounce: 'bounce'
+})
+
```

Comment:
Can you add these options to the table of props in the README?

Reply:
Sure, I will update it on this weekend

Reply:
Have updated the readme